cypress/e2e/signUp.cy.js

```diff
@@ -1,12 +1,36 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+import SignUpPageObject from "../support/pages/signUp.pageObject";
+const signUpPage = new SignUpPageObject();
+import HomePageObject from '../support/pages/home.pageObject';
+const homePage = new HomePageObject();
+const wrongEmail = 'test@gmail.com';
 
 describe('Sign Up page', () => {
-  before(() => {
+  beforeEach(() => {
+    let user;
+    cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = new Object(generateUser);
```

Comment:
The use of `new Object(generateUser)` is unnecessary. You can directly assign `user = generateUser;` since `generateUser` is already an object.